README.md

```diff
@@ -7,188 +7,6 @@ This is the repo for the newly updated Girl Develop It website. In this README y
-
-### Running the tests
-1. Make sure that you've run bundle install to get all the gems required installed
-2. In the root directory of the project run `rspec`
-
-To run the tests in a single file run `rspec spec/#{path_to_file}`
-To run just one test, or one group of tests run `rspec -e 'the name of the test'`
-
-For more information on testing with rspec you may want to check these resources:
-
-General Testing:
-[Unit Testing](http://martinfowler.com/bliki/UnitTest.html)
-[Definition of Terms](https://programmers.stackexchange.com/questions/48237/what-is-an-integration-test-exactly)
-
-Rspec info:
-[Better Specs Website](http://betterspecs.org/)
-[Rspec Documentation](http://rspec.info/)
-
-Testing Ruby on Rails:
-[Everyday Rails Testing Book](https://leanpub.com/everydayrailsrspec)
-[General Overview of Testing Rails Apps](http://robots.thoughtbot.com/how-we-test-rails-applications)
-
-
-### Deploying to Heroku
-
-If you are an owner and/or have been approved by one of the [Project Leads](https://github.com/girldevelopit/gdi-new-site/blob/master/CONTRIBUTORS.md#project-leads), than this section is relevant to you!
-
-1. Be sure that have the [Heroku tool belt](https://toolbelt.heroku.com/) installed.
-2. Add a remote to the GDI Heroku instance with `heroku git:remote -a girl-develop-it`.
-3. Now deploy to Heroku with `git push heroku master`.
-
-**NOTE:** This process will be updated as we start incorporating continuous integration processes, as well as after the site goes officially live.
-
-
-## Troubleshooting
-
-Here are a few things you can do to help if you run into any trouble with the installation process. If you need assistance, join the #website channel in our official Girl Develop It Slack.
-
-
-```sh
-	Is the server running locally and accepting
-	connections on Unix domain socket "/var/pgsql_socket/.s.PGSQL.5432"?
-```
-
-Generally, this error means that you do not have a postgres process started on your machine. Try doing the following steps:
-
-1. Make sure `~/Library/LaunchAgents` does not include `*.plist`.
-2. Restart your computer.
-3. Make sure that there is a postgres process running on your machine.
-4. Add this to your ~/.bash_profile: `export PGHOST=localhost`.
-5. Restart terminal or run `source ~/.bash_profile`.
-6. Try `rake` or `createdb` commands again.
-
-After that, you can run `rails server` to start the server on port `3000` or `rails console` for a REPL.
-
-### Upgrading PostgreSQL from 9.2.*
-
-1. Uninstall PostgreSQL. 
-2. ```sh rm rf /usr/local/var/postgres/ ``` (or wherever you have your postgres/ directory).
-3. Install PostgreSQL 9.3.
+We have migrated our documentation over to Read The Docs! Learn more about how to contribute to our code base by following along here: [https://readthedocs.org/projects/gdi-new-site/](https://readthedocs.org/projects/gdi-new-site/).
```

Comment:
Bah! I wanted the site to point to `gdi-website` and changed the RTFD url to that. Thought it would update the url too, but am thinking the url path has to sync up with the repo name? /me shrugs.

Regardless, good catch friend! :+1: